README.md

```diff
@@ -1,7 +1,6 @@
 # CMakeWrapper
 
-[![Build Status](https://travis-ci.org/JuliaPackaging/CMakeWrapper.jl.svg?branch=master)](https://travis-ci.org/JuliaPackaging/CMakeWrapper.jl)
-[![Build status](https://ci.appveyor.com/api/projects/status/x0k8dy47kau99bhr?svg=true)](https://ci.appveyor.com/project/StevenGJohnson/cmakewrapper-jl)
+![CI](https://github.com/aminya/CMakeWrapper.jl/workflows/CI/badge.svg)
```

Comment:
Can you change the URL back to `JuliaPackaging` here?

Reply:
Fixed in https://github.com/JuliaPackaging/CMakeWrapper.jl/pull/24/commits/ee083e81427fa416c55e893621122e08bb963572